plugins/ocm/src/components/ClusterStatusPage/ClusterStatusPage.tsx

```diff
@@ -147,24 +140,28 @@ const CatalogClusters = () => {
     return <CircularProgress />;
   }
 
-  const data = clusterEntities.map(ce => {
-    return {
-      name: (
-        <EntityRefLink entityRef={ce.entity}>{ce.cluster.name}</EntityRefLink>
-      ),
-      status: <Status status={ce.cluster.status} />,
-      infrastructure: ce.cluster.platform,
-      version: (
-        <Update
-          data={{
-            version: ce.cluster.openshiftVersion,
-            update: ce.cluster.update,
-          }}
-        />
-      ),
-      nodes: <NodeChips nodes={ce.cluster.nodes} />,
-    };
-  });
+  const data = clusterEntities
+    ? clusterEntities.map(ce => {
```

Comment:
I think this would be better:

```suggestion
  const data = clusterEntities?.map(ce => {...
  ...}) || [];
```